quickstarts/microsoft.network/nsg-create-with-diagnostic-logs/main.bicep

```diff
@@ -0,0 +1,100 @@
+@description('Network security group name')
+param nsgName string = uniqueString(resourceGroup().id)
+
+@description('Network security group location')
+param location string = resourceGroup().location
+
+@description('Security rules')
+@metadata({
```

Comment:
is there a doc link we can point to instead - seems like this has the potential to become out-dated quickly and no one will know...

Reply:
updated to contain doc link instead